app/scripts/app/models/variable_exploration/variable_selection.directives.js

```diff
@@ -247,10 +252,14 @@ angular
                 var variable = d.is_group
                   ? Object.assign({ is_group: true }, d.original)
                   : d.original;
+
                 $scope.set_focused_variable(variable);
                 $scope.search.value = null;
                 $scope.search.group = null;
```

Comment:
I think that the apply has to be kept for clicks occurring in the svg. Those are outside angular context, and the digest cycle has then to be forced-